commands/retrieval_client_daemon_test.go

```diff
@@ -52,7 +52,7 @@ func TestSelfDialRetrievalGoodError(t *testing.T) {
 	assert.Error(t, err)
 	var cmdOutBytes []byte
 	w := bytes.NewBuffer(cmdOutBytes)
-	env.GenesisMiner.DumpLastOutput(w)
+	env.GenesisMiner.DumpLastStdErr(w)
```

Comment:
I was thinking instead of using a dump like this we would either just return the Stderr reader directly or even just return the bytes / string. Not 100% which is better. I think what we want is a way to get _just_ the stderr without any extra bits. `DumpLastOutput` is more for debugging than it is for making assertions.

We could also clean this up with the `AssertStdErrContains` (eg: `AssertStdErrContains(t, env.GenesisMiner, expectedErrStr)`)

Reply:
>We could also clean this up with the AssertStdErrContains (eg: AssertStdErrContains(t, env.GenesisMiner, expectedErrStr))

Great minds think alike :)